### ISSUE TITLE
Specifying analysisFlag in eDISH() produces error

### ISSUE THREAD

**Issue by bzkrouse** [ASA-DIA-InteractiveSafetyGraphics/safetyGraphics#92]
Code: 
eDISH(data=adlbc, 
       id_col = "USUBJID",
       value_col = "AVAL", 
       measure_col = "PARAM", 
       visit_col = "VISIT",
       visitn_col = "VISITNUM", 
       studyday_col = "ADY",
       normal_col_low = "A1LO", 
       normal_col_high = "A1HI", 
       measure_values = list(ALT = "Alanine Aminotransferase (U/L)",
                             AST = "Aspartate Aminotransferase (U/L)",
                             TB = "Bilirubin (umol/L)",
                             ALP = "Alkaline Phosphatase (U/L)"),
      analysisFlag = list(value_col = "SAFFL", values = list("Y")),
      debug_js = TRUE)

console:
![image](https://user-images.githubusercontent.com/14199771/50161285-647aaa80-02a9-11e9-983d-d1d2df1436c2.png)

(also tried passing the values as a string, same error)

**Comment by jwildfire** [comment]
Problem here is actually that `baseline` is set to `null` - just need to improve the default there. Will put a little fix in v0.16.1 to deal with that edge case.

**Comment by jwildfire** [comment]
Until v0.16.1 is released, this code runs successfully: 
```
eDISH(data=adlbc,
id_col = "USUBJID",
value_col = "AVAL",
measure_col = "PARAM",
visit_col = "VISIT",
visitn_col = "VISITNUM",
studyday_col = "ADY",
normal_col_low = "A1LO",
normal_col_high = "A1HI",
measure_values = list(ALT = "Alanine Aminotransferase (U/L)",
AST = "Aspartate Aminotransferase (U/L)",
TB = "Bilirubin (umol/L)",
ALP = "Alkaline Phosphatase (U/L)"),
analysisFlag = list(value_col = "SAFFL", values = list("Y")),
baseline = list(values=list()),
debug_js = TRUE)   
```
